core/rpc/src/rpc_impl/transfer.rs

```diff
@@ -29,6 +30,48 @@ where
     C: CkbRpc + Clone + Send + Sync + 'static,
 {
     pub(crate) fn inner_transfer_complete(
+        &self,
+        udt_hash: Option<H256>,
+        from: InnerAccount,
+        items: Vec<InnerTransferItem>,
+        change: Option<String>,
+        fee_rate: u64,
+    ) -> Result<TransactionCompletionResponse> {
+        let mut estimate_fee = START_ESTIMATE_FEE;
+        loop {
+            let TransactionCompletionResponse {
+                tx_view,
+                sigs_entry,
+            } = self.inner_transfer_complete_with_fixed_fee(
```

Comment:
It is not commonly used to directly binding values to the field in the structure, and it is not recommended to use it like this.

Reply:
updated.